dist/index.js

```diff
@@ -104,17 +104,21 @@ var redisStore = function redisStore() {
     },
     get: function get(key, options, cb) {
       return new Promise(function (resolve, reject) {
+        var defaultOpts = { parse: true };
         if (typeof options === 'function') {
           cb = options;
+          options = defaultOpts;
         }
 
+        if (!options) defaultOpts = options;
```

Comment:
I think you might have wanted to have this assignment turned around. Please also add some brackets while you're add it.